normalize/coll.py

```diff
@@ -237,7 +241,7 @@ def coll_to_tuples(cls, coll):
                 i += 1
 
     def itertuples(self):
-        return self._values.iteritems()
+        return iter(self._values.items())
```

Comment:
Should we bring this type into the 3.x world of not using iterator-based accessors?